app/models/user.rb

```diff
@@ -2,7 +2,8 @@ class User < ApplicationRecord
   has_secure_password
   has_many :orders
 
-  validates :first_name, :last_name, :password, presence: true
+  validates :first_name, :last_name, presence: true
+  validates :password, presence: true, allow_nil: true
```

Comment:
Are we testing these new validations on the model somewhere? If not, can we add some tests?

Reply:
The password allowing nil is the only thing that changed - this is so that we are able to change the first name or last name without the pw. I can put another test in the model tomorrow but this instance is being tested in the by allowing the success.